bcda/api/alr_test.go

```diff
@@ -62,7 +62,7 @@ func (s *AlrTestSuite) TestAlrRequest() {
 
 	// Set up request with the correct context scoped values
 	req := httptest.NewRequest("GET",
-		"http://bcda.cms.gov/api/v1/alr/Patient/$export?type=Patient,Observation&_typeFilter=Patient",
+		"http://bcda.cms.gov/api/v1/alr/$export?type=Patient,Observation&_typeFilter=Patient",
```

Comment:
I think we can leave off `type` and `typeFilter` for now - MVP is just basic export of all resource types (iirc, type/typeFilter are leftover from our initial URL construction way back when, and we may need to tweak these once we add in the ability to filter on specific fhir resources)

Reply:
Yea. Let's Remove everything that follows ? for now.

http://bcda.cms.gov/api/v1/alr/$export